source/whats-new.txt

```diff
@@ -28,8 +28,31 @@ Learn what's new in:
 What's New in 4.6
 -----------------
 
-See `v4.6.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.6.0>`__
-on GitHub.
+New features of the 4.6 Node.js driver release include:
+
+TypeScript: ``ChangeStreamDocument``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Ability to define your own custom type for the top level document
+  returned in a ``change`` event.
```

Comment:
Issue: the term is "change event". In the current formatting, it seems like "change" is a type, variable, object, method, collection name, etc. It doesn't appear to be any of these.
Suggestion: remove the formatting for change. If "change" is a type or field, refer to it appropriately and describe what it is.